sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/json/JSONOptions.scala

```diff
@@ -59,6 +62,8 @@ object JSONOptions {
     allowNumericLeadingZeros =
       parameters.get("allowNumericLeadingZeros").map(_.toBoolean).getOrElse(false),
     allowNonNumericNumbers =
-      parameters.get("allowNonNumericNumbers").map(_.toBoolean).getOrElse(true)
+      parameters.get("allowNonNumericNumbers").map(_.toBoolean).getOrElse(true),
+    allowBackslashEscapingAnyCharacter =
+      parameters.get("allowBackslashEscapingAnyCharacter").map(_.toBoolean).getOrElse(true)
```

Comment:
Happy New Year Owen!

At first, I've created PR with false default.

But Xin advised to me that "I'd actually change the default value to true" in previous comment so I've changed.

If you think it doesn't make sense, please call me again. I will change that immediately

Thank you